analyzer/metadata-plugin/pom.xml

```diff
@@ -158,9 +164,9 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.8.1</version>
                 <configuration>
-                    <annotationProcessors>
-                        <annotationProcessor>org.pf4j.processor.ExtensionAnnotationProcessor</annotationProcessor>
```

Comment:
Please do not comment out the PF4J annotation processor in the POM file. If you have problem in Intellij regarding the annotation processor, please check out the last step in the main README file of the project.

Reply:
Got it. Thanks!